src/main/java/org/assertj/core/util/Throwables.java

```diff
@@ -148,6 +150,21 @@ public static Throwable getRootCause(Throwable throwable) {
     return throwable;
   }
 
+  /**
+   * Get the causes chain of a {@link Throwable}.
+   *
+   * @param throwable the {@code Throwable} to get causes chain from.
+   * @return the causes chain, including the {@link Throwable} provided as parameter.
+   */
+  public static List<Throwable> getCausesChain(Throwable throwable) {
```

Comment:
I'm kind of unsettled if the `throwable` in test should be included or not in the list.
On one hand it's nice to have the complete list and assert any of the throwables in the chain, on the other hand, one might say that `getCausesChain` only contains causes down from the "current" level, excluding the current level. Moreover, it's easier to assert the `throwable` in question separately rather than excluding it from the list manually if the user doesn't want to assert it.

I have a slight preference to keep it as is (including the `throwable` in test), but I'd say to specifically mention the inclusive behaviour in the javadocs, here and in `asCausesChain()` method.

Reply:
I tend to favor not having `actual` in the resulting list, starting with its direct cause instead.

Reply:
When I first read `causeChain` I thought it would only contain causes and not the initial exception so I would leave it out.
We can always add a `causeChainWithInitialThrowable` later on if users ask for it.

Reply:
May I propose to introduce both in the same PR? Perhaps one could be `causesChain` and the other one `throwableChain` or `throwableTree`, which includes the initial one.